upp-concept-publishing-provisioner/ansible/decom.yml

```diff
@@ -31,7 +31,7 @@
         msg: "{{concept_pub_stack_output}}"
 
     - name: Delete Secondary Concept Publishing stack
-      when: "{{is_multi_region}} == True"
+      when: ("{{is_multi_region}}" == "true") or ("{{is_multi_region}}" == "True")
```

Comment:
Just a suggestion - it's a bit cleaner if we just lowercase the value of the `is_multi_region` value then match, rather than accepting two explicit options.

That way we catch `true`, `True`, `TRUE`, etc (which is unlikely to ever happen, but we may as well at least trap it). :)

See here for how to do it:
https://serverfault.com/a/701590/428068